lib/pages/chat_draft/draft_chat_view.dart

```diff
@@ -67,190 +67,194 @@ class DraftChatView extends StatelessWidget {
+                                      child: PageTransitionSwitcher(
+                                        transitionBuilder: (
+                                          Widget child,
+                                          Animation<double> primaryAnimation,
+                                          Animation<double> secondaryAnimation,
+                                        ) {
+                                          return SharedAxisTransition(
+                                            animation: primaryAnimation,
+                                            secondaryAnimation:
+                                                secondaryAnimation,
+                                            transitionType:
+                                                SharedAxisTransitionType.scaled,
+                                            fillColor: Colors.transparent,
+                                            child: child,
+                                          );
+                                        },
+                                        child: chatDraftArgument
+                                                    .showEmojiPicker ==
+                                                false
+                                            ? TwakeIconButton(
+                                                paddingAll: DraftChatViewStyle
+                                                    .bottomBarButtonPaddingAll(
+                                                  chatDraftArgument
+                                                      .inputText.isEmpty,
+                                                ),
+                                                tooltip:
+                                                    L10n.of(context)!.emojis,
+                                                onPressed: chatDraftArgument
+                                                    .emojiPickerAction,
+                                                icon: Icons.tag_faces,
+                                              )
+                                            : TwakeIconButton(
+                                                paddingAll: DraftChatViewStyle
+                                                    .bottomBarButtonPaddingAll(
+                                                  chatDraftArgument
+                                                      .inputText.isEmpty,
+                                                ),
+                                                tooltip:
+                                                    L10n.of(context)!.keyboard,
+                                                onPressed: () =>
+                                                    chatDraftArgument.inputFocus
+                                                        ?.requestFocus(),
+                                                icon: Icons.keyboard,
+                                              ),
```

Comment:
- Create a `Widget _xyzButtonBuilder()`
```suggestion
  Widget _xyzButtonBuilder(
    BuildContext context,
    bool isEmojiPicker,
  ) {
    return TwakeIconButton(
      paddingAll: ChatAppBarTitleStyle.paddingXyz(
        chatDraftArgument.inputText.isEmpty,
      ),
      tooltip:
          isEmojiPicker ? L10n.of(context)!.keyboard : L10n.of(context)!.emojis,
      onPressed: isEmojiPicker
          ? () => chatDraftArgument.inputFocus?.requestFocus()
          : chatDraftArgument.emojiPickerAction,
      icon: isEmojiPicker ? Icons.keyboard : Icons.tag_faces,
    );
  }
```

Reply:
done